### ISSUE TITLE
timings format in slots

### ISSUE THREAD

**Issue by parikshitjuneja22** [dyte-submissions/vit-hiring-2023-phase1-test#3]
The timings format for slots in the API reference doc is in ISO string format but in the data.py it is in 12 hr format (8:00 AM).

**Comment by Tanmay000009** [comment]
Yes. This is is an issue.  Can you please clarify, which one should we use in final submission?

**Comment by feniljain** [comment]
Hey, 

Thanks for reporting this!

We mean 12 hour format here :)

Keeping this open for anyone who comes across with same doubt
